tox.ini

```diff
@@ -7,6 +8,7 @@ deps=
     pytest
     pytest-cov
     mock
+    redis==2.10.6
```

Comment:
Do we need this now after enabling `requirements.txt`?

Reply:
This is for local tox testing. Yes.